roles/dnsmasq/tasks/main.yml

```diff
@@ -61,5 +61,10 @@
 
 - name: disable resolv.conf modification by dhclient
   copy: src=dhclient_nodnsupdate dest=/etc/dhcp/dhclient-enter-hooks.d/nodnsupdate mode=u+x backup=yes
+  when: ansible_os_family == "Debian"
+
+- name: disable resolv.conf modification by dhclient
```

Comment:
is it possible to disable also resolvconf package that manage the resolv.conf on ubuntu (default pkg)?

Either disable it, or write the conf in one of those files and trigger an update (I prefer this)
       /etc/resolvconf/resolv.conf.d/base
       /etc/resolvconf/resolv.conf.d/head
       /etc/resolvconf/resolv.conf.d/tail
              
http://manpages.ubuntu.com/manpages/lucid/man8/resolvconf.8.html